src/Main/index.js

```diff
@@ -28,11 +30,18 @@ export const Main = () => {
     };
     setTodos([...todos, newTodo]);
   };
+
+  const handleChangeTodo = (id) => {
+    const newTodos = [...todos];
+    const todo = newTodos.find((duetodo) => duetodo.id === id);
+    todo.done = !todo.done;
+    setTodos(newTodos);
+  };
   return (
     <main className="App-main">
-      <Todos todos={todos} />
+      <Todos todos={todos} changeTodo={handleChangeTodo} />
       <hr />
-      <NewToDoForm addTodo={handleAddTodo} />
+      <NewToDoForm addTodo={handleAddTodo} changeTodo={handleChangeTodo} />
```

Comment:
`changeTodo={handleChangeTodo}` not required by form, please remove this attribute